frontend/src/metabase/core/components/FormField/FormField.styled.tsx

```diff
@@ -90,7 +101,7 @@ export const FieldRoot = styled.div<FieldRootProps>`
     props.alignment === "end" &&
     props.orientation === "horizontal" &&
     "space-between"};
-  margin-bottom: 1.25rem;
+  margin-bottom: ${props => (props.noMargin ? 0 : "1.25rem")};
```

Comment:
Without having to create a new one-off styled component every time we only use a single field in a section, introducing this new flag would make it easier. I justify this because it seems a typical use case to me.

#### Before this flag
<img width="359" alt="image" src="https://user-images.githubusercontent.com/1937582/231552898-9829de0f-6129-41db-a8e5-6afa82eea190.png">


#### With this flag
<img width="361" alt="image" src="https://user-images.githubusercontent.com/1937582/231552847-30dbb805-814e-41df-9ec9-5e072eee4728.png">